f2/src/forms/FinalFeedbackForm.js

```diff
@@ -0,0 +1,194 @@
+          }
+        }}
+        validate={validate}
+        render={({ handleSubmit, values }) => (
+          <Stack
+            as="form"
+            onSubmit={handleSubmit}
+            shouldWrapChildren
+            spacing={12}
+          >
+            <Control as="fieldset" name="wasServiceHard">
+              <FormLabel as="legend" htmlFor="wasServiceHard" mb={2}>
+                <Trans id="finalFeedback.wasServiceHard.label" />
+              </FormLabel>
+              <Stack spacing={4} shouldWrapChildren>
+                {wasServiceHard.map(key => {
+                  return (
+                    <Box key={key}>
+                      <RadioButtonArrayControl
+                        name="wasServiceHard"
+                        value={key}
+                      >
+                        {i18n._(key)}
+                      </RadioButtonArrayControl>
+                    </Box>
+                  )
+                })}
+              </Stack>
+            </Control>
+
+            <Control as="fieldset" name="wouldYouUseAgain">
+              <FormLabel as="legend" htmlFor="wouldYouUseAgain" mb={2}>
+                <Trans id="finalFeedback.wouldYouUseAgain.label" />
+              </FormLabel>
+              <Stack spacing={4} shouldWrapChildren>
+                {wouldYouUseAgain.map(key => {
+                  return (
+                    <Box key={key}>
+                      <RadioButtonArrayControl
+                        name="wouldYouUseAgain"
+                        value={key}
+                      >
+                        {i18n._(key)}
+                      </RadioButtonArrayControl>
+                    </Box>
+                  )
+                })}
+              </Stack>
+            </Control>
+
+            <Field name="howCanWeDoBetter">
+              {props => (
+                <FormControl>
+                  <FormLabel htmlFor="howCanWeDoBetter">
+                    <Trans id="finalFeedback.howCanWeDoBetter.label" />
+                  </FormLabel>
+                  <FormHelperText>
+                    <Trans id="finalFeedback.howCanWeDoBetter.helper">
+                      <VisuallyHidden as="span" />
+                    </Trans>
+                  </FormHelperText>
+                  <TextArea
+                    id="howCanWeDoBetter"
+                    name={props.input.name}
+                    value={props.input.value}
+                    onChange={props.input.onChange}
+                  />
+                </FormControl>
+              )}
+            </Field>
+
+            {showWarning ? (
+              <Control>
+                <Alert status="warning">
+                  <AlertIcon />
+                  <Trans id="finalFeedback.warning" />
+                </Alert>
+              </Control>
+            ) : null}
+
+            <Button type="submit" mb={10}>
+              <Trans id="landingPage.nextButton.reportNow" />
+            </Button>
```

Comment:
Missing a translation here. Also make the button blue!
```suggestion 
            <Button
              type="submit"
              w={{ base: '100%', md: 'auto' }}
              variantColor="blue"
            >
              <Trans id="midFeedback.submit" />
            </Button>

Reply:
Good catch!